include/pluginlib/class_list_macros.h

```diff
@@ -39,31 +39,12 @@
 
 #include <class_loader/class_loader.h>
 
+/// Register a class with class loader to effectively export it for plugin loading later.
 /**
- * @macro This version was deprecated in favor of PLUGINLIB_DECLARE_CLASS
- * @param - class_name - An alias for the class (no special characters allowed)  (IGNORED AS OF PLUGINLIB 1.9)
- * @param - class_type - The real class name with namespace qualifier (e.g. Animals::Lion)
- * @param - base_class_type - The real base class type from which class_type inherits
- */
-#define PLUGINLIB_REGISTER_CLASS(class_name, class_type, base_class_type) \
-  CLASS_LOADER_REGISTER_CLASS_WITH_MESSAGE(class_type, base_class_type, "In file " __FILE__ " pluginlib WARNING: PLUGINLIB_REGISTER_CLASS is deprecated, please use PLUGINLIB_EXPORT_CLASS instead. You can run the script 'plugin_macro_update' provided with pluginlib in your package source folder to automatically and recursively update legacy macros. Base = base_class_type, Derived = derived_class_type")
-
-/**
- * @macro This version is the most in use and requires package name in addition to fields in PLUGINLIB_REGISTER_CLASS
- * @param - pkg - The package that exports the plugin (IGNORED AS OF PLUGINLIB 1.9)
- * @param - class_name - An alias for the class (no special characters allowed)  (IGNORED AS OF PLUGINLIB 1.9)
- * @param - class_type - The real class name with namespace qualifier (e.g. Animals::Lion)
- * @param - base_class_type - The real base class type from which class_type inherits
- */
-#define PLUGINLIB_DECLARE_CLASS(pkg, class_name, class_type, base_class_type) \
-  CLASS_LOADER_REGISTER_CLASS_WITH_MESSAGE(class_type, base_class_type, "pluginlib WARNING: In file " __FILE__ " PLUGINLIB_DECLARE_CLASS is deprecated, please use PLUGINLIB_EXPORT_CLASS instead. You can run the script 'plugin_macro_update' provided with pluginlib in your package source folder to automatically and recursively update legacy macros.  Base = base_class_type, Derived = derived_class_type")
-
-/**
- * @macro This version was only made possible with pluginlib 1.9 series. It's the easiest to use and now the official way of exporting classes.
- * @param - class_type - The real class name with namespace qualifier (e.g. Animals::Lion)
- * @param - base_class_type - The real base class type from which class_type inherits
+ * \param class_type The real class name with namespace qualifier (e.g. Animals::Lion)
+ * \param base_class_type The real base class type from which class_type inherits
```

Comment:
Nit: I'd rather keep the documentation markup consistent within the repo. We can open a follow up PR to change the syntax in all pluginlib files at once to use the doxygen `\` style

Reply:
Oh, sorry, this slipped in (I have layers of changes 😄). I'll revert this part of this pr.

Reply:
7d10fff, but I left the restructuring of the docs because the content changed on purpose. This should be consistent enough with the existing docs to not stand out I think.

Reply:
now worries, thanks for the quick fix